python/cudf/setup.py

```diff
@@ -93,6 +93,11 @@
         "cudf._lib.arrow": ["*.pxd"],
         "cudf._libxx": ["*.pxd"],
         "cudf._libxx.cpp": ["*.pxd"],
+        "cudf._libxx.cpp.column": ["*.pxd"],
+        "cudf._libxx.cpp.io": ["*.pxd"],
+        "cudf._libxx.cpp.scalar": ["*.pxd"],
+        "cudf._libxx.cpp.table": ["*.pxd"],
+        "cudf._libxx.cpp.wrappers": ["*.pxd"],
```

Comment:
Could you check if a wildcard works here instead of specifying each folder? I.E. could we do `cudf._libxx.cpp.*`?

Reply:
It works using a wildcard, done ;)